DNN Platform/Website/App_GlobalResources/GlobalResources.resx

```diff
@@ -1428,4 +1428,25 @@ Sincerely,
   <data name="cc_message.Text" xml:space="preserve">
     <value>This website uses cookies to ensure you get the best experience on our website.</value>
   </data>
-</root>
+  <data name="Administrators.Role" xml:space="preserve">
+    <value>Administrators 1</value>
+  </data>
+  <data name="All Users.Role" xml:space="preserve">
+    <value>Everyone 1</value>
+  </data>
+  <data name="Superuser.Role" xml:space="preserve">
+    <value>SuperUsers 1</value>
+  </data>
+  <data name="Unauthenticated Users.Role" xml:space="preserve">
+    <value>Unauthenticated Users 1</value>
+  </data>
+  <data name="Registered Users.Role" xml:space="preserve">
+    <value>Registered Users 1</value>
+  </data>
+  <data name="Unverified Users.Role" xml:space="preserve">
+    <value>Unverified Users 1</value>
+  </data>
+  <data name="Subscribers.Role" xml:space="preserve">
+    <value>Subscribers 1</value>
+  </data>
```

Comment:
Does this mean every language pack that already translated these global roles will need to be updated?

I assume these aren't support to have ` 1` at the end?
```suggestion
  <data name="Administrators.Role" xml:space="preserve">
    <value>Administrators</value>
  </data>
  <data name="All Users.Role" xml:space="preserve">
    <value>Everyone</value>
  </data>
  <data name="Superuser.Role" xml:space="preserve">
    <value>SuperUsers</value>
  </data>
  <data name="Unauthenticated Users.Role" xml:space="preserve">
    <value>Unauthenticated Users</value>
  </data>
  <data name="Registered Users.Role" xml:space="preserve">
    <value>Registered Users</value>
  </data>
  <data name="Unverified Users.Role" xml:space="preserve">
    <value>Unverified Users</value>
  </data>
  <data name="Subscribers.Role" xml:space="preserve">
    <value>Subscribers</value>
  </data>
```

Reply:
@bdukes Peter used the extra 1 just for testing for now he told me. (to see the difference)

Reply:
I will do this when I remove the WIP from the title. Otherwise I can't verify whether it's working.